src/main/java/lesson4/RestAssuredWorker.java

```diff
@@ -0,0 +1,66 @@
+package lesson4;
+
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.apache.http.HttpHeaders;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RestAssuredWorker {
```

Comment:
this class has common functionality with ApacheWorker. Make appropriate changes to re-use the codebase

Reply:
Created a separate class and method for common functionality